### PR TITLE
random option added to custom starters

### PR DESCRIPTION
**Pull request by Zach-Meadows**
Tested for FR/LG and HG/SS should be fine for the rest of the games as well.

- random option is now default selection, when choosing custom starters
- choosing random for each slot will randomize that slot only

### CODE REVIEW

**Review by Ajarmar — CHANGES_REQUESTED** [review]
Looks good. There are just a few changes I'd like to see before I can merge this:

1. I don't think the default being random is intuitive for the custom starter setting, so please change that back to how it was before, Instead, please add a short blurb to the custom starter tooltip mentioning that it's possible to set starters to random. (Modify [this line](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/blob/55f42ba61f4d9b5eb20c79c0bd54fb1eb909ba6a/src/com/dabomstew/pkrandom/newgui/Bundle.properties#L94))
2. I think there should be a safeguard to prevent the random Pokemon from picking any of the Pokemon that you've specified. While adding the picked starters to the `pickedStarters` list prevents the starters after that from picking that same starter, it does not handle the case where a random starter occurs before a specified one. Easiest way of handling this is probably just to go through the `customStarters` array and add any specified Pokemon to the `banned` list before picking any Pokemon.
3. I think it looks a bit weird to have "RANDOM" in all caps since it's just a word rather than a Pokemon name - it would look better as "Random".

**Comment by Zach-Meadows** [comment]
Changes made and committed. 

Let me know if there's anything else i need to do!

Thanks

File: src/com/dabomstew/pkrandom/romhandlers/AbstractRomHandler.java [hunk 1]
```diff
@@ -3182,25 +3182,67 @@ public void metronomeOnlyMode() {
...
+            banned.addAll(abilityDependentFormes);
+        }
+        if (banIrregularAltFormes) {
+            banned.addAll(getIrregularFormes());
+        }
+        // loop to add chosen pokemon to banned, preventing it from being a random option.
+        for (int i = 0; i < customStarters.length; i = i + 1){
+            if (!(customStarters[i] - 1 == 0)){
+                System.out.println(customStarters[i] - 1);
+                System.out.println(romPokemon.get(customStarters[i] - 1));
```
Comment (Ajarmar) [inline src/com/dabomstew/pkrandom/romhandlers/AbstractRomHandler.java:3208]: Seems like you left some debug prints here. Other than that, everything looks good now

Comment (Zach-Meadows) [reply src/com/dabomstew/pkrandom/romhandlers/AbstractRomHandler.java]: resolved! thanks

**Review by Ajarmar — APPROVED** [review]
Great! I'll merge it, thanks for the contribution

**Pull request merged**